bin/reth/src/commands/import.rs

```diff
@@ -145,6 +160,13 @@ impl ImportCommand {
             // override the tip
             let tip = file_client.tip().expect("file client has no tip");
             info!(target: "reth::cli", "Chain file chunk read");
+            // this will be start header of next chunk
+            let tip_header = file_client.tip_header().unwrap();
+
+            if start_header.is_none() {
+                // start header is genesis block
+                start_header = file_client.start_header();
+            }
```

Comment:
genesis block should be imported out of pipeline, we should query file client for it

Reply:
hm, I'm not sure if that is possible, check this test out which I based myself on

https://github.com/paradigmxyz/reth/blob/6863cdb42bbac29de57b66739c8e0fc7b4d5dbaa/crates/net/downloaders/src/file_client.rs#L488

Reply:
the test is correct because in the test the first header was set as local head